src/utils/index.ts

```diff
@@ -95,3 +95,13 @@ export function isInvalidEmail(email: string) {
 
     return !EMAIL_RE.test(email);
 }
+
+export function isContributor(contributor: unknown): contributor is Contributor {
+    if (!contributor) {
+        return false;
+    }
+
+    const {login, name, email} = contributor as Contributor;
+
+    return Boolean(login || name || email);
```

Comment:
if all of them are empty - isn't it an empty Contributor type?

Reply:
changed it on `in` clause